editor/src/components/canvas/ui/floating-insert-menu.tsx

```diff
@@ -297,13 +299,7 @@ function useComponentSelectorStyles(): StylesConfig<InsertMenuItem, false> {
       groupHeading: (styles): CSSObject => {
         return {
           // ...styles,
-          color: colorTheme.fg7.value,
-          height: 25,
-          right: 12,
-          position: 'absolute',
-          display: 'flex',
-          alignItems: 'center',
-          pointerEvents: 'none',
+          display: 'none',
```

Comment:
I'm assuming this was unintentionally committed? I don't believe we want to remove the group headings.

Reply:
I think we do! the reason is that group headings are something that react-select gives us, but the way we want the insert menu to look like it just doesn't work as a heading (since we want a clickable element in every row). So to make this happen, I removed the group heading via the css here, and added the same label via `CustomOption`

Reply:
there might be betters ways to do this, I think this is the most straightforward and leads to the fewest surprises